src/components/DappBrowser/BrowserTab.tsx

```diff
@@ -253,6 +248,94 @@ export const BrowserTab = React.memo(
       };
     });
 
+    const animatedGestureHandlerHeight = useDerivedValue(() => {
+      // For some reason driving the WebView height with a separate derived
+      // value results in slightly less tearing when the height animates
+      const animatedIsActiveTab = animatedActiveTabIndex?.value === animatedTabIndex.value;
+      const closeButtonHeight = getCloseTabButtonHeight({ animatedMultipleTabsOpen: animatedMultipleTabsOpen.value });
+
+      if (!animatedIsActiveTab) return COLLAPSED_WEBVIEW_HEIGHT_UNSCALED - closeButtonHeight;
+
+      const progress = tabViewProgress?.value || 0;
+
+      return interpolate(
+        progress,
+        [0, 100],
+        [
+          animatedIsActiveTab ? 0 : COLLAPSED_WEBVIEW_HEIGHT_UNSCALED - closeButtonHeight,
+          COLLAPSED_WEBVIEW_HEIGHT_UNSCALED - closeButtonHeight,
+        ],
```

Comment:
I think this can return early and skip a bunch of unnecessary work, right?

```
if (!tabViewVisible?.value?) {
  return { display: 'none' }
}
```

or `{ opacity: 0, pointerEvents: 'none' }` if `display: none` doesn't work for some reason

Reply:
Instead of adjusting its `zIndex`

Reply:
yeah great point